### DISCUSSION TITLE
Dell S2722QC - brightness slider position and actual monitor brightness do not match

### DISCUSSION THREAD

**Discussion by stormbringer83** [MonitorControl/MonitorControl#1121] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

1. Volume control works, but volume slider position in the menu bar is not updated when I change volume level on the monitor itself using physical buttons. E.g.: volume slider set to 0, monitor volume set to 0. I set volume using monitor buttons to 50% - slider remains at 0.

2. Same problem with brightness: brightness slider position is not updated when I change brightness level on the monitor itself using physical buttons.

3. Brightness slider position and actual monitor brightness do not match. Any slider position between 0 and 50% corresponds to monitor brightness 0; slider position 50% corresponds to monitor brightness 0; slider position 75% corresponds to monitor brightness 50%; slider position 100% corresponds to monitor brightness 100%.

### Expected behavior

1. Volume and brightness sliders position in the menu bar are updated when volume & brightness are changed on the monitor using physical buttons.
2. Brightness slider position in the menu bar matches actual monitor brightness, i.e. left position is 0% brightness, right position is 100% actual monitor brightness.

### Environment Information (please complete the following information)

```markdown
- macOS version: 12.4
- Mac model: MacBook Air (M1, 2020)
- MonitorControl version: 4.1.0 Build 7034
- Monitor(s): Dell S2722QC
- Monitor Cable(s)/Connection(s): USB-C
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
Hi there, this is all normal and expected. The app does not continuously poll the display settings via DDC (that would wreak havoc with many displays). You can set up the app to at least read settings on startup or after sleep but that is disabled by default since many displays are not compatible with this.

Brigtness 0-50% is software dimming and 50-100% is hardware (DDC) backlight control by default. You can change this in settings or adjust the transition point in Advanced Settings under Displays.
